optimizely/decision_service.py

```diff
@@ -34,20 +35,27 @@ def __init__(self, config, user_profile_service):
     self.config = config
     self.logger = config.logger
 
-  @staticmethod
-  def _get_bucketing_id(user_id, attributes):
+  def _get_bucketing_id(self, user_id, attributes):
     """ Helper method to determine bucketing ID for the user.
 
     Args:
       user_id: ID for user.
       attributes: Dict representing user attributes. May consist of bucketing ID to be used.
 
     Returns:
-      String representing bucketing ID for the user. Fallback to user's ID if not provided.
+      String representing bucketing ID if it is a String type in attributes else return user ID.
```

Comment:
@mikeng13 Do we also accept empty string as bucketing ID?

Reply:
Empty string is a valid user ID right now so it makes sense to allow it for bucketing ID as well.